### PR TITLE
Fix typo in project settings doc

### PR DESCRIPTION
**Pull request by RouganStriker**

### CODE REVIEW

File: source/includes/project_settings.md [hunk 1]
```diff
@@ -124,10 +124,10 @@ Content-Type: application/json
                     "id": "Q21",
                     "text": "Is this a question?",
                     "description": "Help text",
-                    "format": "MC" // SC for single choice, else MC
+                    "format": "MC", // SC for single choice, else MC
                     "mandatory": false,
                     "satisfied": true,
-                    "subquestions": [...]
+                    "subquestions": [...],
```
Comment (geoffwhittington) [inline source/includes/project_settings.md]: Its not really clear, are sub-questions of the same format as `questions`? If there are no subquestions do we output null or an empty list?

Comment (funkaoshi) [inline source/includes/project_settings.md]: We should have a full example here.

Comment (RouganStriker) [inline source/includes/project_settings.md]: Included subquestions in the example

**Comment by funkaoshi** [comment]
Good stuff @RouganStriker

**Pull request merged**